public/app/features/explore/Logs/LogsTable.tsx

```diff
@@ -14,38 +15,42 @@ import {
   ValueLinkConfig,
 } from '@grafana/data';
 import { config } from '@grafana/runtime';
-import { Table } from '@grafana/ui';
+import { AdHocFilterItem, Table } from '@grafana/ui';
+import { FILTER_FOR_OPERATOR, FILTER_OUT_OPERATOR } from '@grafana/ui/src/components/Table/types';
 import { separateVisibleFields } from 'app/features/logs/components/logParser';
-import { parseLogsFrame } from 'app/features/logs/logsFrame';
+import { LogsFrame, parseLogsFrame } from 'app/features/logs/logsFrame';
 
 import { getFieldLinksForExplore } from '../utils/links';
 
+import { fieldNameMeta } from './LogsTableWrap';
+
 interface Props {
   logsFrames?: DataFrame[];
```

Comment:
In `LogsTableWrap` this is required and that is the only place that uses this component, so we can make it required as well.

Reply:
Should be fixed now!